src/providers/info.rs

```diff
@@ -211,6 +221,17 @@ impl<C: Cache> DefaultInfoProvider<'_, C> {
         }
     }
 
+    async fn load_chain_info(&self, chain_id: &str) -> ApiResult<Option<ChainInfo>> {
+        let url = format!("{}/v1/chains/{}/", base_config_service_url(), chain_id);
+        let data = RequestCached::new(url)
+            // .cache_duration(safe_info_cache_duration())
+            // .error_cache_duration(short_error_duration())
+            // .request_timeout(safe_info_request_timeout())
+            .execute(self.client, self.cache)
```

Comment:
Do we want custom cache duration (and error duration) and request timeouts for this? I would also add a new variant to the `InvalidationPattern::Chains` so we can more easily debug this.

Reply:
Yes I would say so